packages/substrate-client/src/chainhead/public-types.ts

```diff
@@ -86,7 +82,7 @@ export interface FollowResponse {
     StorageResponse
   >
   header: (hash: string) => Promise<string>
-  unpin: (...hashes: Array<string>) => Promise<void>
+  unpin: (hashes: Array<string>) => Promise<void>
```

Comment:
This is probably the most significant breaking change. While testing, I realized that 99.99% of the times we will be calling this method with an Array of hashes anyways... So, the spread looked fancy, but in reality is quite impractical.